lib/connect_four.rb

```diff
@@ -63,13 +70,13 @@ def play
     puts @board.format_board
 
     until @game.game_over == true
-      @board.update_board_human(@game.get_human_selection)
+      @board.update_board(@game.get_human_selection, :human)
```

Comment:
Can we pass current player here?

Reply:
done! I had to add a few extra lines in the play method to switch the current player back and forth -- this is hopefully temporary, and will be refactored out in the next issue (#26).